BREAKING.md

```diff
@@ -28,6 +29,9 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - [Minor event naming correction on IFluidContainerEvents](#IFluidContainerEvents-event-naming-correction)
 - [Add assertion that prevents sending op while processing another op](#add-assertion-that-prevents-sending-op-while-processing-another-op)
 
+### Update `existing` field in runtime
+The existing flag was updated to be required in runtime. The existing field in IContainerContext has also been removed and isn't used to check for an existing snapshot.
```

Comment:
I think we need to add @deprecated annotation in main, to help people move away from these API sooner (and not get blocked with next is released)